hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/UpgradeUtilities.java

```diff
@@ -464,7 +464,7 @@ public static void createDataNodeVersionFile(File[] parent,
     for (int i = 0; i < parent.length; i++) {
       File versionFile = new File(parent[i], "VERSION");
       StorageDirectory sd = new StorageDirectory(parent[i].getParentFile());
-      storage.createStorageID(sd);
+      storage.createStorageID(sd, false);
```

Comment:
I don't remember adding this. Besides its a test file so I don't care as much ;-)